app-map.js

```diff
@@ -1,5 +1,7 @@
 steal("can/util", "can/map", "can/compute", function(can){
 
+	var CanMap = can.Map;
+
```

Comment:
No reason other than having a local pointer, probably should have added this to the arguments.